src/cli/evaluations/schedule.py

```diff
@@ -169,11 +169,15 @@ def job_factory() -> str:
     all_configurations = generate_configurations(Path(config_path))
 
     # Then, we can run the training, passing parameters as required
+    source_bucket_prefix = "source"
```

Comment:
In this case, I would just add the "source" to the code_location path since we are not expect anyone to change it.

Reply:
I will add a parameter to control the code_location path